web/index.html

```diff
@@ -717,24 +715,25 @@ <h4>Discussions</h4>
       </li>
       <li id="callout-2">
         One may suggest that we could also stop generation in S+F as soon as a correct solution is
-        found. But due to the probablistic nature of sampling, which cannot guarantee replication
+        found. However due to the probabilistic nature of sampling, which cannot guarantee replication
         across successive generations, we need to collect the full set of samples for every problem
-        in order to properly estimate the pass@k metrics for various values of k.
+        to properly estimate the pass@k metrics for various values of k.
+        We could alternatively sample until success and calculate the empirical expected k to compare with the average rollout of MCTS, but we suffice with estimating the pass@k metric.
       </li>
     </ol>
     <h2>Extending the experiment to VerilogEval</h2>
     <p>
       Using MCTS on the HumanEval dataset proved quite successful, outperforming the ~equivalent S+F
       5-sample baseline by a decent margin and even the 20-sample baseline at much less compute
-      cost. But due to the raw strength of
+      cost. However due to the raw strength of
```

Comment:
Again, think it's fine to keep "but" here in favour of "however". I see that grammarly has a specific preference for always using however, it's mostly a stylistic point but not a big fan of it personally